frontend/src/FullPageRoutes/AccountsPage.tsx

```diff
@@ -0,0 +1,263 @@
+import * as React from 'react';
+import { Grid, makeStyles, Button, Typography, TextField, InputAdornment, IconButton, Divider, FormControl, FormControlLabel, FormLabel, RadioGroup, Radio, FormHelperText, Dialog, DialogContent, DialogActions} from '@material-ui/core';
+import EditIcon from '@material-ui/icons/Edit';
+import { useState, useEffect } from 'react'
+
+export type userAccounts={
+    firstName: string,
+    lastName: string,
+    username: string,
+    password: string,
+    email: string,
+    successor: string,
+}
+
+const AccountValue = {
+    firstName:'',
+    lastName: '',
+    username: '',
+    password: '',
+    email: '',
+    successor: '',
+}
+
+
+const useStyles = makeStyles({
+    root:{
+        '& .MuiFormControl-root':{
+            width: '100%',
+            margin: 8,
+        },
+    },
+    topBar: {
+        margin: 1,
+        padding: 50,
+        backgroundColor: "rgb(238,235,228)",
+    },
+    pageContent: {
+        margin: 8,
+        padding: 110,
+    },
+    button:{
+        margin: 5,
+    },
+})
+
+
+export default function AccountsPage(): JSX.Element {
+    const classes = useStyles();
+    const [editMode, editModeOn] = useState(false);
+    const [value, setValues] = useState('');
+    const [btnText, setBtnText] = useState('Delete or Deactivate Account')
+    const [open, setOpen] = useState(false)
+    const [accounts, setAccounts] = useState( )
+    
+       
+
+    const handleChange = () => {
+
+    }
+    
+    const handleEdit = () => {
+        editModeOn(true)
+        
+    }
+
+    const handleSave = () => {
+
+    }
+
+    const handleCancel = () => {
+        editModeOn(false)
+    }
+
+    const handleDeleteChanges = (event: any) =>{
+        setValues(event.target.value);
+        
+        
+        
+    }
+
+    const handleAccountChanges = (event: any) => {
+        event.preventDefault();
+
+        if (value === 'Deactivating Account'){
+            setBtnText('Deactivate Account');
+            setOpen(true);
+
+        } else if (value === 'Delete Account'){
+            setBtnText('Delete Account');
+            setOpen(true)
+        }
+        else{
+            setBtnText('Delete or Deactivate Account')
+            setOpen(false);
+        }
+    }
+
+    
+    const handleClose = () =>{
+        setOpen(false);
+    }
+
+    useEffect(()=>{
+        React.request({ path: "/api/profile", method: "get" })
```

Comment:
Since we've decided to make Account its own page, I think we will need to separate the account request from profile since the pages will not be using all the same user data.